src/main/resources/files/liberty/TestSecureController.java.tpl

```diff
@@ -48,7 +48,7 @@ public class TestSecureController {
         String jwt = generateJWT(key);
 
         // any method to send a REST request with an appropriate header will work of course.
-        WebTarget target = ClientBuilder.newClient().target("http://localhost:8281/demo/data/protected");
+        WebTarget target = ClientBuilder.newClient().target("http://localhost:8281/[# th:text="${artifact_id}"/]/data/protected");
```

Comment:
@rdebusscher Cannot we parameterize the URL and have just one TestSecureController template? IMHO this is te only LIberty specific line, right...?

Reply:
Or we can have a look at how we deploy app on context root with Liberty.  That would be easier ....

Reply:
Indeed. 
I have no idea :)    @Emily-Jiang ?

Reply:
Leave it for now. I'll do a PR to properly tidy up the pom.xml and hope to set context root to /. I will then remove the duplication.